rotkehlchen/tests/api/test_historical_balances.py

```diff
@@ -0,0 +1,198 @@
+from http import HTTPStatus
+from typing import TYPE_CHECKING
+
+import pytest
+import requests
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.constants import DAY_IN_SECONDS
+from rotkehlchen.constants.assets import A_BTC, A_ETH
+from rotkehlchen.db.history_events import DBHistoryEvents
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.base import HistoryEvent
+from rotkehlchen.history.events.structures.types import (
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.tests.utils.api import (
+    api_url_for,
+    assert_error_response,
+    assert_ok_async_response,
+    assert_proper_sync_response_with_result,
+    wait_for_async_task,
+)
+from rotkehlchen.types import Location, Timestamp
+from rotkehlchen.utils.misc import ts_sec_to_ms
+
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
+    from rotkehlchen.db.dbhandler import DBHandler
+
+START_TS = Timestamp(1672531200)
+
+
+def _create_test_events(db: 'DBHandler') -> None:
```

Comment:
No I did not mean this. I meant to remove this function and instead keep the fixture. Just to move this function's code in the fixture, and use the fixture in the tests

Reply:
okay, reverted that.